presentation/src/main/java/com/anytypeio/anytype/presentation/relations/value/tagstatus/TagOrStatusValueViewModel.kt

```diff
@@ -333,6 +353,7 @@ class TagOrStatusValueViewModel(
                 success = {
                     dispatcher.send(it)
                     sendAnalyticsRelationValueEvent(analytics)
+                    emitCommand(command = Command.Dismiss, delay = 300L)
```

Comment:
consider if possible extracting this delay into constant with naming explaining why we need this delay

Reply:
added, because it is more convenient to show the user that the selection worked before closing the menu.

Reply:
done